gdrive

```diff
@@ -84,53 +80,63 @@ def search_filename(service, file_name=None, page_size=10, list_all=False, page_
         query = "name contains '%s'" % file_name
     if page_token is None:
         page_token = ''
-    fields = 'nextPageToken, files/id, files/name, files/size, files/modifiedTime, files/owners'
+    fields = ('nextPageToken, files/id, files/name, files/size, '
+              'files/modifiedTime, files/modifiedByMeTime, files/owners')
     search = service.files().list(q=query,
-                                  orderBy='modifiedTime desc',
+                                  orderBy='modifiedByMeTime desc',
                                   fields=fields,
                                   pageSize=page_size,
                                   pageToken=page_token).execute()
     return search.get('files', []), search.get('nextPageToken', None)
 
 def list_files(service, file_name, list_all=False):
     """List files present on user's Google Drive"""
-    files, next_page = search_filename(service, file_name, 10, list_all)
-    for file in files:
-        describe_file(file)
+    files_found, next_page = search_filename(service, file_name, 10, list_all)
+    for metadata in files_found:
+        describe_file(metadata)
     while next_page is not None:
         if input(">>> Press Enter for the next results\n"):
             break
-        files, next_page = search_filename(service, file_name, 10, next_page)
-        for file in files:
-            describe_file(file)
+        files_found, next_page = search_filename(service, file_name, 10, list_all, next_page)
+        for metadata in files_found:
+            describe_file(metadata)
 
 def describe_file(file):
```

Comment:
Now you take in a metadata. Rename

Reply:
Renamed the 'file' variable to 'metadata'